src/modules/Builder/components/Item/Item.tsx

```diff
@@ -64,7 +64,7 @@ export const Item = ({
             <StyledActivityDescription>{description}</StyledActivityDescription>
           </>
         )}
-        {count && (
+        {count !== undefined && (
```

Comment:
is this check really need? Per my understanding, count is a number and we should always show the next component, right?